contracts/src/v0.8/keystone/CapabilityRegistry.sol

```diff
@@ -74,6 +74,16 @@ contract CapabilityRegistry is OwnerIsCreator, TypeAndVersionInterface {
   /// exists.
   error CapabilityAlreadyExists();
 
+  /// @notice This error is thrown when a capability with the provided ID is
+  /// not found.
+  /// @param capabilityId The ID used for the lookup.
+  error CapabilityDoesNotExist(bytes32 capabilityId);
+
+  /// @notice This error is thrown when trying to deprecate a capability that
+  /// is already deprecated.
+  /// @param capabilityId The ID of the capability that is already deprecated.
+  error CapabilityIsDeprecated(bytes32 capabilityId);
```

Comment:
This name is very similar to the event name - maybe rename to "CapabilityAlreadyDeprecated"? Or even consider not throwing an error at all?

Reply:
Good shout, renamed 👍